app/pages/patients/patients.js

```diff
@@ -62,9 +63,14 @@ var Patients = React.createClass({
     var noPatientsSetupStorage = this.renderNoPatientsSetupStorageLink();
     var patients = this.renderPatients();
 
+    var bgClasses = cx({
```

Comment:
Around here, 'bg' means blood glucose. Can you choose another name? Maybe `bckgrndClasses`? I dunno. Anything but 'bg' ;)